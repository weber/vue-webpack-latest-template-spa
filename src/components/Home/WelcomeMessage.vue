<template>
  <div>
    <h1>привет сучка!!!</h1>
    <h2>привет сучка!</h2>
    <h3>привет сучка!</h3>
    <h4>привет сучка!</h4>
    <p>привет сучка!</p>
    {{ name }}
  </div>
</template>

<script>
export default {
  props: {
    name: {
      type: String,
      required: true
    }
  },
  methods: {
    dfdfd () {
    }
  }
}
</script>
