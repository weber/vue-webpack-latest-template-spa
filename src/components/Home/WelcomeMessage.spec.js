import { shallow } from '@vue/test-utils'
import WelcomeMessage from '@components/Home/WelcomeMessage'

describe('WelcomeMessage.vue', () => {
  it('renders props.name when passed', () => {
    const name = 'tester'

    const wrapper = shallow(WelcomeMessage, {
      propsData: { name }
    })

    // expect(wrapper.text()).toBe('привет сучка! привет сучка! привет сучка! привет сучка! привет сучка!')

    expect(wrapper.html()).toMatchSnapshot()
  })
})
