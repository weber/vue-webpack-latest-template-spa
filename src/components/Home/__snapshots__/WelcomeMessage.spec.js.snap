// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WelcomeMessage.vue renders props.msg when passed 1`] = `
<div>
  <h1>привет сучка!</h1>
  <h2>привет сучка!</h2>
  <h3>привет сучка!</h3>
  <h4>привет сучка!</h4>
  <p>привет сучка!</p>
</div>
`;
